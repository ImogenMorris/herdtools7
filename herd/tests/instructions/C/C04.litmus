C C04
(* Tests 128-bit types *)
{ }

P0 (int128_t *x, __int128_t *y, uint128_t *z, __uint128_t *a) {
  int128_t r0 = *x;
  __int128_t r1 = *y;
  uint128_t r2 = *z;
  __uint128_t r3 = *a;
}

forall (0:r0 = 0 /\ 0:r1 = 0 /\ 0:r2 = 0 /\ 0:r3  = 0)
