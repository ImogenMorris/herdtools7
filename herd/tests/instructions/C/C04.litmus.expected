Test C04 Required
States 1
0:r0=0x0; 0:r1=0x0; 0:r2=0x0; 0:r3=0x0;
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall (0:r0=0x0 /\ 0:r1=0x0 /\ 0:r2=0x0 /\ 0:r3=0x0)
Observation C04 Always 1 0
Hash=65d4730a05131206025f681ebb855dbc

